daprdocs/content/en/reference/components-reference/supported-name-resolution/nr-kubernetes.md

```diff
@@ -7,15 +7,36 @@ description: Detailed information on the Kubernetes DNS name resolution componen
 
 ## Configuration format
 
-Kubernetes DNS name resolution is configured automatically in [Kubernetes mode]({{< ref kubernetes >}}) by Dapr. There is no configuration needed to use Kubernetes DNS as your name resolution provider.
+Generally, Kubernetes DNS name resolution is configured automatically in [Kubernetes mode]({{< ref kubernetes >}}) by Dapr. There is no configuration needed to use Kubernetes DNS as your name resolution provider unless there are some overrides necessary for the Kubernetes name resolution component.
+
+In the scenario that an override is required, within a [Dapr Configuration]({{< ref configuration-overview.md >}}) CRD, add a `nameResolution` spec and set the `component` field set to `"kubernetes"`. Then other configuration fields can be set as needed in a `configuration` map as seen below.
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: appconfig
+spec:
+  nameResolution:
+    component: "kubernetes"
+    configuration:
+      "clusterDomain": "cluster.local"  # Mutually exclusive with the template field
+      "template": "{{.ID}}-{{.Data.region}}.internal:{{.Port}}" # Mutually exclusive with the clusterDomain field
+```
 
 ## Behaviour
 
 The component resolves target apps by using the Kubernetes cluster's DNS provider. You can learn more in the [Kubernetes docs](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/).
 
 ## Spec configuration fields
 
-Not applicable, as Kubernetes DNS is configured by Dapr when running in Kubernetes mode.
+The configuration spec is fixed to v1.3.0 of the Consul API
+
+| Field        | Required | Type | Details  | Examples |
+|--------------|:--------:|-----:|:---------|----------|
+| clusterDomain       | N        | `string` | The cluster domain to be used for resolved addresses. This field is mutually exclusive with the `template` file.| `cluster.local`
+| template | N        | `string` | A template string to be parsed when addresses are resolved using [text/template](https://pkg.go.dev/text/template#Template) . The template will be populated by the fields in the [ResolveRequest](https://github.com/dapr/components-contrib/blob/v1.12.0-rc.3/nameresolution/requests.go#L20) struct. This field is mutually exclusive with `clusterDomain` field. | `{{.ID}}-{{.Data.region}}.{{.Namespace}}.internal:{{.Port}}`
```

Comment:
Rather than hardcoding a version, I think there's a shortcode in Hugo we can use to get the latest Dapr version here (or at least the branch). We do that in the Event Grid component

Reply:
I am not seeing this in azure event grid. @dapr/maintainers-docs Is there a way to point to code branch or version dynamically?

Reply:
My bad, it was in the Cloudflare Workers component: https://github.com/dapr/docs/blob/0accc68dee289a992a73d454f14712066a549a0d/daprdocs/content/en/reference/components-reference/supported-state-stores/setup-cloudflare-workerskv.md?plain=1#L159-L160